io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVInstanceCollection.java

```diff
@@ -85,8 +85,21 @@ private void proceedToNext() {
 
 			// initialize reader if necessary
 			if (csvReader == null) {
-				boolean skipFirst = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_FIRST_LINE)
-						.as(Boolean.class, false);
+
+				int skipN = 0;
+				Boolean skipType = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES)
+						.as(boolean.class);
+
+				if (skipType == null) {
+					skipN = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES)
+							.as(Integer.class, 0);
+				}
+				else if (skipType == true) {
```

Comment:
Although, `if (skipType == true)` is completely acceptable, you can also shorten it to `if (skipType)`